src/vs/workbench/contrib/mergeEditor/browser/view/colors.ts

```diff
@@ -14,7 +14,7 @@ export const diff = registerColor(
 
 export const diffWord = registerColor(
 	'mergeEditor.change.word.background',
-	{ dark: '#9bb9551e', light: '#9bb9551e', hcDark: '#9bb9551e', hcLight: '#9bb9551e', },
+	{ dark: '#9ccc2c33', light: '#9ccc2c66', hcDark: '#9ccc2c66', hcLight: '#9ccc2c66', },
```

Comment:
Should dark and hcDark match in this case?

Reply:
Eventually, no. I'm doing a separate pass on the hc colors. We're kind of using dark and light colors for their hc counterparts across the board until then.

Reply:
Oh, sorry. I misunderstood. You are right—fixed it.